response/slack/views.py

```diff
@@ -50,7 +50,7 @@ def slash_command(request):
             Text(
                 label="Report",
```

Comment:
I think we should change this label as well, as it's not immediately clear what it means. I'd suggest `Title`